tools/integration_tests/util/client/storage_client.go

```diff
@@ -123,6 +128,23 @@ func ReadChunkFromGCS(ctx context.Context, client *storage.Client, object string
 	return string(content), nil
 }
 
+func NewWriterWithSupportForZB(ctx context.Context, o *storage.ObjectHandle, client *storage.Client) (wc *storage.Writer, err error) {
+	var attrs *storage.BucketAttrs
+	wc = o.NewWriter(ctx)
+	attrs, err = client.Bucket(o.BucketName()).Attrs(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get attributes for bucket %q: %w", o.BucketName(), err)
+	}
+	if attrs.StorageClass == "RAPID" {
+		if setup.IsZonalBucketRun() {
```

Comment:
I think this if-else condition is unnecessary.  If storage class is rapid than we should believe it as zonal only.